exercises/octal/src/test/java/OctalTest.java

```diff
@@ -1,49 +1,99 @@
 import org.junit.Test;
 import org.junit.Ignore;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-
-import java.util.Arrays;
-import java.util.Collection;
 
 import static org.junit.Assert.assertEquals;
 
-@RunWith(Parameterized.class)
 public class OctalTest {
+    private Octal octal;
+
+    @Test
+    public void testOne() {
```

Comment:
These test names could maybe describe what the test does a bit better. Looking at this it's easy to read it as `the first test` instead of `test converting the number one`, if that makes sense? So maybe something like `testCovertDecimalOneToOctal` or `testConvertOneToOctal` would be more descriptive? 🙂